mln/services/message.py

```diff
@@ -49,7 +49,20 @@ def open_message(user, message_id):
 	message.save()
 	return message
 
-def send_message(user, recipient_id, body_id):
-	"""Send a message to someone."""
+def create_message(user, recipient_id, body_id): 
+	"""Creates a message in-memory. To actually send, use [send_message]."""
 	_check_recipient(user, recipient_id)
-	return Message.objects.create(sender=user, recipient_id=recipient_id, body_id=body_id)
+	return Message(sender=user, recipient_id=recipient_id, body_id=body_id)
+
+def send_message(message, attachment):
+	"""Send a message to someone."""
+	if not message.recipient.profile.is_networker:
+		message.save()
+		if attachment is not None: attachment.save()
+	# Send the networker's response
+	for trigger in NetworkerMessageTrigger.objects.filter(networker=message.recipient): 
+		if not trigger.evaluate(message, attachment): continue
+		trigger.send_message(message.sender)
+		break
+	else:  # networker doesn't have a trigger for this message
+		Message.objects.create(sender=message.recipient, recipient=message.sender, body_id=MLNError.I_DONT_GET_IT)

```

Comment:
MLNError is actually a separate category in MLN's data xml, and the "I don't get it" message is not one of them. Just add the ID as a separate constant instead.